src/reporting/ReportingWindow.sol

```diff
@@ -224,6 +225,10 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable, IReportingWi
         return numberOfReportsByMarket[_market];
     }
 
+    function hasReceivedAllReport(IMarket _market) public afterInitialized constant returns (bool) {
+        return receivedAllReport[_market];
```

Comment:
Hmm, I think I misunderstood your question in #dev earlier but get it now.  Since any market in all reporting phase necessarily has reports (since it received some in limited reporting phase) we need to decide if we will blindly accept the limited report outcome should no one report during all reporting.  I think this situation further cements my desire to count the dispute bond as negative reports.

I suppose even if we did have the dispute bond be negative, we can't change the tentative outcome to the "second best" because we don't know who the second best was...

We should probably have a more involved discussion on how we want to move forward with this code (dealing with dispute bonds as negative stake and no reporters during all reporting).

Reply:
I suspected this would provoke more discussion :]

I'll address the easy bits here and modify it to use a distinct phase for this case in the mean time still.